crates/trigger/src/key_value.rs

```diff
@@ -0,0 +1,41 @@
+use std::{path::Path, sync::Arc};
```

Comment:
@dicej Please take another look. In going back to `change_dir_all` here I ended up moving this into a new file and refactoring a bit.